src/app/modules/ix-forms/services/form-error-handler.service.ts

```diff
@@ -53,8 +55,9 @@ export class FormErrorHandlerService {
       const errorMessage = extraItem[1];
 
       let control = this.getFormField(formGroup, field, fieldsMap);
+      const controlNames = this.ixFormService.getControls().map((ctrl) => ctrl.name);
```

Comment:
Let's move this into a new method in the form service, e.g. `getControlsNames` or update an existing method to return it.